gallery/visualtests/nxDropdown.js

```diff
@@ -39,13 +41,22 @@ describe('NxDropdown', function() {
       await button.click();
     });
 
-    it('has a dark blue button border with expanded menu', async function() {
+    it('has a blue inner outline when focused with expanded menu', async function() {
       const [targetElement] = await waitAndGetElements(defaultSelector);
 
       await moveMouseAway();
 
       await checkScreenshot(targetElement, 251, 376);
     });
+
+    it('has a dark grey border when not foucsed with expanded menu', async function() {
```

Comment:
```suggestion
    it('has a dark grey border when not focused with expanded menu', async function() {
```

might also need to rename the visual test file name!

Reply:
> might also need to rename the visual test file name!

Thanks for the catch, renamed here [9acc493](https://github.com/sonatype/sonatype-react-shared-components/pull/995/commits/9acc493d3a5f4f3fd9e992bcaa41ff1e38e8ce45)